src/components/CardSection.js

```diff
@@ -0,0 +1,34 @@
+import searchIcon from "../assets/Search.svg";
+import CardList from "./CardList";
+import "./CardSection.css";
+
+export default function CardSection({ cardList }) {
+  console.log(cardList);
+  return (
+    <div className="CardSection">
+      <div className="cardFrame">
+        <div className="searchBar">
+          <div className="searchBarElement">
+            <img id="searchIcon" src={searchIcon} alt="검색 아이콘"></img>
+            <input id="inputText" placeholder="링크를 검색해보세요."></input>
```

Comment:
이렇게 self-closing으로 써주시면 좋을 것 같아요~
```suggestion
            <img id="searchIcon" src={searchIcon} alt="검색 아이콘" />
            <input id="inputText" placeholder="링크를 검색해보세요." />
```